riff-raff.yaml

```diff
@@ -0,0 +1,21 @@
+stacks:
+- discussion
+deployments:
+  static:
+    parameters:
+      cacheControl:
+      - pattern: .*\.js$
+        value: public, max-age=315360000
+      - pattern: .*\.map$
+        value: public, max-age=315360000
+      - pattern: .*
+        value: public, max-age=60
+      mimeTypes:
+        js: application/javascript
+        json: application/json
+        map: text/plain
+      prefixStack: false
```

Comment:
Ah wow, that's super annoying. I'll talk to Simon, but looks like you're approach is best for now!